ecl/hqlcpp/hqlsource.cpp

```diff
@@ -1802,13 +1803,15 @@ ABoundActivity * SourceBuilder::buildActivity(BuildCtx & ctx, IHqlExpression * e
     translator.gatherActiveCursors(ctx, parentCursors);
 
     bool isSpill = tableExpr && tableExpr->hasAttribute(_spill_Atom);
+    if (isSpill && (activityKind == TAKdiskread))
+        activityKind = TAKspillread;
```

Comment:
Under what circumstances will there be a spilled read where activityKind != TAKdiskread.  Why is this necessary?

Reply:
diskcounts, aggregates and normalize

Reply:
disk counts, aggregates, normalize